integration/grpc_test.go

```diff
@@ -0,0 +1,134 @@
+package integration
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"net"
+	"os"
+	"time"
+
+	"github.com/containous/traefik/integration/helloworld"
+	"github.com/containous/traefik/integration/try"
+	"github.com/go-check/check"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+var LocalhostCert = []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
I think having all certs in a shared directory would be beneficial. This would prevent having to have people generate certs, and we could have wildcard certs/single host certs/ etc all in one easy to use convenient place for testing.